### PR TITLE
Add INSTALL with how-to use this product

### PR DESCRIPTION
**Pull request by andr2000**
Add instructions on how to build this product with
the use of the Xen-troops build scripts.

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Our new build system uses set of additional meta layers and tools.
+1. Please, clone the following build scripts, master branch:
+
+git clone https://github.com/xen-troops/build-scripts.git
+cd build-scripts
+
+2. In the build-scripts directory you will find a sample configuration
+file xt-build-server.cfg:
+
+cp xt-build-server.cfg xt-gen3-test.cfg
```
Comment (aanisov) [inline INSTALL]: xt-build-server.cfg refers to a private repository instead xen-troops:
 
> \# URI of the build-manifest repository to use
> xt_manifest_uri = https://github.com/andr2000/meta-xt-products.git

File: INSTALL [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+ - workspace_storage_base_dir: change it to where downloads and other files will be put
+
+For example,
+
+workspace_base_dir = /tmp/workspace_base
+workspace_storage_base_dir = /tmp/workspace_storage_base
+
+4. Run the build script:
+
+python ./build_prod.py --build-type dailybuild --machine salvator-x-h3 --product gen3-test --with-local-conf --config xt-gen3-test.cfg
```
Comment (aanisov) [inline INSTALL]: Any specific python version to be used?

Comment (aanisov) [inline INSTALL]: On my machine running a command results in:

> Traceback (most recent call last):
>  File "./build_prod.py", line 2, in <module>
>    import git
> ImportError: No module named git

Comment (aanisov) [inline INSTALL]: Solved with 
> sudo apt-get install python-git

It worth to be specified.

File: INSTALL [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+5. After that you will have all the build environment setup at workspace_base_dir
+directory.
+
+6. Download and copy EVA proprietary archives on your local filesystem in some directory:
+ - R-Car_Gen3_Series_Evaluation_Software_Package_for_Linux-20170427.zip
+ - R-Car_Gen3_Series_Evaluation_Software_Package_of_Linux_Drivers-20170427.zip
+
+7. Edit local.conf:
+
+workspace_base/build/local.conf
```
Comment (aanisov) [inline INSTALL]: wrong path, must be `workspace_base/build/conf/local.conf`

File: INSTALL (hunk 1 above)
Comment (andr2000) [inline INSTALL]: will do with a separate commit

File: INSTALL (hunk 2 above)
Comment (andr2000) [inline INSTALL]: 2.7? but I am not sure to state in the document

Comment (andr2000) [inline INSTALL]: done

File: INSTALL (hunk 3 above)
Comment (andr2000) [inline INSTALL]: done
